packages/zoe/src/zoe.js

```diff
@@ -175,11 +175,7 @@ import { makeTables } from './state';
 /**
  * @callback MakeContract The type exported from a Zoe contract
```

Comment:
s/MakeContract/makeContract/

Reply:
No, this is meant to be capitalized. I think this is also out of scope for this PR, since I didn't change it.

Reply:
This PR is updating the documentation for MakeContract, which is what makes it relevant. 

I don't think TypeScript will recognize `MakeContract` as the declaration that pertains to the function that must be named `makeContract`.  Are you saying that it should be capitalized because that's the style supported by TypeScript?

Hmm.  I see that automaticRefund explicitly declares that its `makeContract` is of type `MakeContract`, but none of the other contracts do that, which leaves VSCode intuiting the type from the actual behavior, which isn't very helpful for developers writing new contracts.  When I'm writing up recommended contract style guidelines, I may follow automaticRefund, since that provides better guidance to devs.

Reply:
I still think it is out of scope since I didn't change it. But, all of the callbacks including this one are capitalized.

Reply:
I'm not trying to say that it should be fixed in this PR. I'm trying to figure out whether it's doing the right thing, since this PR is changing the declaration of `MakeContract`.

The other `@callback`s that are capitalized are defining types that  are used as `@property` declarations within an interface. `ContractFacet` has a bunch of those. That's not the case with this one. 

The way ContractFacet uses them is to declare
```js
/**
 * @typedef {Object} ContractFacet
 * @property {Reallocate} reallocate Propose a reallocation of extents per offer
...
*
 * @callback Reallocate
 * @param  {OfferHandle[]} offerHandles An array of offerHandles
 * @param  {AmountKeywordRecord[]} newAllocations An
 * @param  {Keyword[]} sparseKeywords An array of string
 * @returns {TODO}
*/
```

This defines `Reallocate` to be a callback with these  params and return value, then uses that to declare `reallocate` to be a method within `ContractFacet`. This means when you hover over `reallocate` in a context that's known to be a `ContractFacet`, VSCode can display details about the `Reallocate` declaration. 

In the case of `MakeContract`, there's no enclosing class with a declaration of `makeContract`, so nothing connects the function name that appears in the code (which a developer might want documentation for) to the declaration of `MakeContract`.

Reply:
Ok, thanks for the analysis! I'm going to create an issue from your comment